subprojects/common/test-okhttp/src/main/kotlin/com/avito/test/http/MockDispatcher.kt

```diff
@@ -14,11 +12,8 @@ import java.util.Collections
 
 public class MockDispatcher(
     private val unmockedResponse: MockResponse = MockResponse().setResponseCode(418).setBody("Not mocked"),
-    loggerFactory: LoggerFactory
```

Comment:
Discussable. We discuss that with @eugene-krivobokov and consider that if someone need debug info we should make it more obvious i.e. add a fun for that info